src/librustc_mir/monomorphize/collector.rs

```diff
@@ -650,19 +650,24 @@ impl<'a, 'tcx> MirVisitor<'tcx> for MirNeighborCollector<'a, 'tcx> {
         self.super_terminator_kind(block, kind, location);
     }
 
-    fn visit_static(&mut self,
-                    static_: &mir::Static<'tcx>,
+    fn visit_place(&mut self,
+                    place: &mir::Place<'tcx>,
                     context: mir::visit::PlaceContext<'tcx>,
                     location: Location) {
-        debug!("visiting static {:?} @ {:?}", static_.def_id, location);
+        match place {
+            mir::Place::Base(mir::PlaceBase::Static(static_)) => {
```

Comment:
You still need to skip the code below if `promoted.is_some()`

Reply:
Yeah, my bad. Caught it after `git push`. Fixing them. Hope it's fine otherwise.